### PR TITLE
page 1 of Monday issue

### PR DESCRIPTION
**Pull request by haileyensor**
Deliverable 1

### CODE REVIEW

File: 1906-11-26.xml [hunk 1]
```diff
@@ -0,0 +1,1540 @@
...
+                                <placeName>Hamburg</placeName> bound for
+                                <placeName>Beyrout</placeName>.</p>
+                        <p><name>S.S. Lemnos</name>
+                            <date when="1905-07-31">July 31</date> from
+                                <placeName>Hamburg</placeName> bound for
+                                <placeName>Beyrout</placeName>.</p>
+                        <p>For tariff and particulars apply to ADOLPHE STROSS, Alexandria,
+                            Agent.</p>
+                        <p><measure type="indexNo">15-2-905</measure></p>
+                        <div type="advert" xml:id="deg-ad-kml01">
```
Comment (whanley) [inline 1906-11-26.xml]: Insert a `</div>` tag closing the previous ad.

File: 1906-11-26.xml [hunk 2]
```diff
@@ -0,0 +1,1540 @@
...
+                </bibl>
+            </sourceDesc>
+        </fileDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="page" n="1">
+                <div type="masthead">
+                    <head>THE EGYPTIAN GAZETTE</head>
+                    <!-- Issue number 7620, November 26, 1906, 6 pages-->
```
Comment (whanley) [inline 1906-11-26.xml]: enter this information in the content of the next line, rather than in a comment.

File: 1906-11-26.xml [hunk 3]
```diff
@@ -0,0 +1,1635 @@
...
+                            direct and quickest means of <lb/>communication from Egypt to Europe,
+                            North and <lb/>South America, East, South and West Africa, <lb/>India,
+                            Australia, New Zealand, China and Japan.</p>
+                        <p>To secure quick transmission, telegrams should <lb/>be marked <hi>Via
+                                Eastern.</hi></p>
+                        <p>For latest average time to London, see daily <lb/>bulletin in this
+                            paper.</p>
+                        <p>STATIONS IN EGYPT: Alexandria, Cairo, <lb/>Suez, Port-Tewfik, Port-Saïd,
+                            Suakin. Head <lb/>Office. London.</p>
+                        <div type="advert" xml:id="deg-ad-sfp01">
```
Comment (whanley) [inline 1906-11-26.xml]: You need a `</div>` tag to close the ad that precedes this one.

File: 1906-11-26.xml [hunk 4]
```diff
@@ -0,0 +1,1635 @@
...
+                                <cell>Leave Alexandria</cell>
+                                <cell><date when="1906-10-23">8 November</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1906-10-27">13 November</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">10 &amp; 24 November</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">27 November
```
Comment (whanley) [inline 1906-11-26.xml]: the `</div>` tag on this line closes page 2 as soon as you start it--move the tag to the end of page 2, i.e. just before `<div type="page" n="3">`

**Comment by haileyensor** [comment]
hi,

I went through each advertisement and changed all the dates, so they should all be correct now.

Hailey Ensor

On Feb 2, 2018, at 4:01 PM, Will Hanley <notifications@github.com<mailto:notifications@github.com>> wrote:


@whanley requested changes on this pull request.

Good job--almost done deliverable 1 and good start on deliverable 2. I've made a few comments. Also, please run the //date/@when xpath query to find dates that you haven't updated.

________________________________

In 1906-11-26.xml<https://github.com/dig-eg-gaz/content/pull/585#discussion_r165758514>:

> +                    <head>THE EGYPTIAN GAZETTE</head>
+                    <head type="sub">No. <measure quantity="7620">7,620</measure>] ALEXANDRIA,
+                        MONDAY, NOVEMBER 26, 1906. [SIX PAGES P.T. 1. </head>
+                    <div type="advert" xml:id="deg-ad-etc01">
+                        <head>The Eastern Telegraph Company, Limited.</head>
+                        <p>This Company's system of submarine telegraph <lb/>cables is the most
+                            direct and quickest means of <lb/>communication from Egypt to Europe,
+                            North and <lb/>South America, East, South and West Africa, <lb/>India,
+                            Australia, New Zealand, China and Japan.</p>
+                        <p>To secure quick transmission, telegrams should <lb/>be marked <hi>Via
+                                Eastern.</hi></p>
+                        <p>For latest average time to London, see daily <lb/>bulletin in this
+                            paper.</p>
+                        <p>STATIONS IN EGYPT: Alexandria, Cairo, <lb/>Suez, Port-Tewfik, Port-Saïd,
+                            Suakin. Head <lb/>Office. London.</p>
+                        <div type="advert" xml:id="deg-ad-sfp01">


You need a </div> tag to close the ad that precedes this one.

________________________________

In 1906-11-26.xml<https://github.com/dig-eg-gaz/content/pull/585#discussion_r165758756>:

> +                        <p>Hamburg-Amerika Linie.</p>
+                        <p>Return to Egypt<lb/> by the new S.S. Oceana of 9000 Tons, 350 berths
+                            including 90 single cabins, greatest comfort, best cuisine.</p>
+                        <p>FARES: Genoa to Alexandria from £14. Naples to Alexandria from £10.</p>
+                        <table>
+                            <row>
+                                <cell>Leave Alexandria</cell>
+                                <cell><date when="1906-10-23">8 November</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1906-10-27">13 November</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">10 &amp; 24 November</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">27 November


the </div> tag on this line closes page 2 as soon as you start it--move the tag to the end of page 2, i.e. just before <div type="page" n="3">

File: 1906-11-26.xml [hunk 5]
```diff
@@ -0,0 +1,1635 @@
...
+                        <p>From Alexandria</p>
+                        <table rows="12" cols="5">
+                            <head><hi rend="bold">Sailing from Alexandria in November,
+                                1906.</hi></head>
+                            <row>
+                                <cell cols="5"><hi rend="bold">For Marseilles direct</hi></cell>
+                            </row>
+                            <row>
+                                <cell>Friday</cell>
+                                <cell><date when="1905-07-14">30 Nov</date></cell>
```
Comment (whanley) [inline 1906-11-26.xml]: all of the dates in this ad are from 1905!

File: 1906-11-26.xml [hunk 6]
```diff
@@ -0,0 +1,1934 @@
...
+                            </row>
+                        </table>
+                        <table rend="frame" xml:id="SailingfromPortSaid">
+                            <head><hi rend="bold">Sailing from Port Said in November,
+                                1906</hi></head>
+                            <row>
+                                <cell rows="5">For Marseilles Direct</cell>
+                                <cell>Probably on</cell>
+                                <cell>Thursday</cell>
+                                <cell><date when="1905-07-06">8 Nov</date></cell>
```
Comment (whanley) [inline 1906-11-26.xml]: this date and a bunch that follow in this ad need to be updated--in this case to when="1906-11-08"

File: 1906-11-26.xml [hunk 7]
```diff
@@ -0,0 +1,1934 @@
...
+                                <cell>Leave Alexandria</cell>
+                                <cell><date when="1906-10-23">8 November</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1906-10-27">13 November</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">10 &amp; 24 November</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-10">27 November
```
Comment (whanley) [inline 1906-11-26.xml]: this date and those that follow in this ad need to be updated.

File: 1906-11-26.xml [hunk 8]
```diff
@@ -0,0 +1,1934 @@
...
+                            Through freight rates to Inland towns in Great Britain also to the
+                            U.S.A</p>
+                        <table rows="4" cols="5">
+                            <row>
+                                <cell>Westcott <name>S.S. Joshua Nicholson</name></cell>
+                                <cell>expected from</cell>
+                                <cell><placeName>Antwerp</placeName>, <placeName>London</placeName>
+                                    &amp; <placeName>Malta</placeName></cell>
+                                <cell>is due on or about</cell>
+                                <cell><date when="1905-07-16">Nov 26</date></cell>
```
Comment (whanley) [inline 1906-11-26.xml]: this date and those that follow need to be updated.

File: 1906-11-26.xml (hunk 7 above)
Comment (whanley) [inline 1906-11-26.xml]: Move this `</div>` tag, which closes the page, to the end of the page.

Comment (whanley) [inline 1906-11-26.xml]: There is a template for this item. https://dig-eg-gaz.github.io/templates/#egyptian-gazette-subscriptions-and-advertisements

Comment (whanley) [inline 1906-11-26.xml]: Here and afterwards, the problem is that your text is not wrapped in `<div>` tags and in `<p>` tags. All text in xml must be wrapped in one or both of these tags. I can help you with this in class or in office hours.

**Comment by whanley** [comment]
Hailey, this document is not valid (there's a red box), and I can't figure out your deliverable one until it's valid. It looks like you're pasting OCR text into the text view, rather than the author view, and this is causing problems. If you paste it into author view, it will automatically add `<p>` tags. We can help you with this in class.

File: 1906-11-26.xml [hunk 9]
```diff
@@ -0,0 +1,3364 @@
...
+                                <cell>Leave Alexandria</cell>
+                                <cell><date when="1906-10-23">8 November</date> p.m.</cell>
+                                <cell>arr. Naples</cell>
+                                <cell><date when="1906-10-27">13 November</date> a.m.</cell>
+                            </row>
+                            <row>
+                                <cell>,, ,,</cell>
+                                <cell><date when="1906-11-06">10 &amp; 24 November</date> ,,</cell>
+                                <cell>,, ,,</cell>
+                                <cell><date when="19
```
Comment (whanley) [inline 1906-11-26.xml]: what's this line for?

Comment (whanley) [inline 1906-11-26.xml]: this article on Baghdad Railway is a separate item.

Comment (whanley) [reply 1906-11-26.xml]: The OCRed text needs to be corrected.

Comment (whanley) [inline 1906-11-26.xml]: shouldn't this be page 3 (n="3")?

**Comment by whanley** [comment]
@haileyensor let's sit down and try to work out your div issues. Can you make it to Friday office hours?

**Comment by haileyensor** [comment]
Hi,

I can't make it to Friday office hours because I work however I can come in to office hours today.

Hailey Ensor



On Feb 21, 2018, at 12:42 AM, Will Hanley <notifications@github.com<mailto:notifications@github.com>> wrote:


@haileyensor<https://github.com/haileyensor> let's sit down and try to work out your div issues. Can you make it to Friday office hours?

**Comment by whanley** [comment]
Monday looking much better. Break up divs at end of page 2. Where is page 4?

Tuesday and Wednesda: lots of formatting and OCR correcting to do. Maybe you'd be more efficient, in future issues, to paste in the OCR text div by div, rather than page by page?

**Comment by whanley** [comment]
Hailey, lots of great work here, but we've got to sit down together and work through your divs. Come to office hours.

**Pull request merged**